### DISCUSSION TITLE
Dual Screen on Orange Pi 5 Plus

### DISCUSSION THREAD

**Discussion by thanhtantran** [Joshua-Riek/ubuntu-rockchip#207] [General]
Hi,

Can I have dual screen with the version for Orange Pi 5 Plus? How to enable it?

**Comment by Joshua-Riek** [comment]
Yes, you can use both hdmi ports at the same time. There is no need to enable anything.

**Comment by DROM4** [comment]
I haven't been able to get dual screen with my Orange Pi 5b. I use the Ubuntu version I got from here, everything looks good and works fine, except extended monitors. I'm using it right now to write this comment. No multiple screens and the manufacturer says it supports up to 4

**Reply by Joshua-Riek** [reply]
The Orange Pi 5B should support two displays only, one with HDMI and one with USB-C. If you are using some sort of splitter it will not work. The underlying vop2 display pane driver has a lot of issues, especially with USB-C displays.

**Reply by DROM4** [reply]
I have to use an adapter (HDMI to USBC), but I'm not using a splitter. I might be one of the unlucky ones then, because it doesn't work. I was looking into another adapter HDMI to USB for the second monitor, but I don't think that's gonna work either, after what you said. Thanks for the help thou. Appreciate it.
